src/Contact/Contact.tsx

```diff
@@ -175,7 +177,7 @@ export class Contact extends Component {
 												/>
 											</FormControl>
 
-											<FormControl fullWidth required={true}>
+											<FormControl fullWidth required>
```

Comment:
This is indeed better, as long as you are 100% certain that `FormControl` interprets `required` the same as `required={true}`